ecosystem/indexer-grpc/indexer-grpc-utils/src/file_store_operator/mod.rs

```diff
@@ -42,6 +43,15 @@ pub struct FileStoreMetadata {
     pub version: u64,
 }
 
+/// FileStoreMetadata is the metadata for the file store.
+/// It's a JSON file with name: metadata.json.
+#[derive(Serialize, Deserialize, Copy, Clone, Debug)]
+pub struct VerificationMetadata {
+    pub chain_id: u64,
+    // The current version of the file store.
+    pub next_version_to_verify: u64,
```

Comment:
how fast is the verification? is it a problem that we now expect the files-store-operator process to be stateful since it now writes a file? vs previously when we just wrote to a separate file-store? Can we keep this metadata in the file store itself?

Reply:
1. it's like several thousands tps.
2. file processor is a stateful since we have `metadata.json` currently. 
3. yes, it looks like this now:

![image](https://github.com/aptos-labs/aptos-core/assets/112209412/a296340b-e1ee-4840-ba17-969be3250b47)

Reply:
ah I missed the part where you're reusing the file store operator, so the verification state is also persisted into the file store. ignore!